src/Microsoft.Data/BaseColumn.cs

```diff
@@ -163,6 +163,23 @@ public void SetName(string newName, DataFrame dataFrame = null)
         /// </summary>
         public virtual double Median() => throw new NotImplementedException();
 
+        /// <summary>
+        /// Returns a DataFrame with a concise summary of this column
+        /// </summary>
+        public virtual DataFrame Info()
+        {
+            DataFrame ret = new DataFrame();
+            StringColumn infoColumn = new StringColumn("Info", 0);
```

Comment:
You know you are going to append 2 values to these columns, you should initialize them with `length: 2`.